selfdrive/manager.py

```diff
@@ -294,6 +315,7 @@ def prepare_managed_process(p):
       cloudlog.warning("building %s failed, make clean" % (proc, ))
```

Comment:
Might want to add another try except here and return the exception. I don't ever run into these types of errors however